python/bot.py

```diff
@@ -55,6 +55,17 @@ def user_is_ignored(self, user):
             return True
         return False
 
+    def user_is_hero(self, user):
+        try:
+            user_roles = [role.id for role in user.roles]
+        except AttributeError:
+            return False
+        community_hero_role = self.config['admin_roles']
```

Comment:
should be hero_role?

Reply:
Yes, copied the function from `user_is_admin` and forgot to change the config role.